movo_7dof_moveit_config/launch/ompl-chomp_planning_pipeline.launch.xml

```diff
@@ -0,0 +1,19 @@
+<launch>
+  <!-- load OMPL planning pipeline, but add the CHOMP planning adapter. -->
+  <include file="$(find panda_moveit_config)/launch/ompl_planning_pipeline.launch.xml">
```

Comment:
why were these launch files added? also just fyi ur using the config for the Panda arm which is a different robotic arm  make by FrankaEmika not Kinova

Reply:
@bdschnapp